src/router_core/agent_router.c

```diff
@@ -233,6 +235,13 @@ static void qdr_agent_write_column_CT(qd_composed_field_t *body, int col, qdr_co
         qd_compose_insert_uint(body, core->links_blocked);
         break;
 
+    case QDR_ROUTER_UPTIME_SECONDS:
+        if (core->uptime_ticks== 0)
+            qd_compose_insert_uint(body, 0);
```

Comment:
This if-then-else doesn't seem to be any different from simply calling qs_compose_insert_uint.